src/Options/RbacGuardOptions.php

```diff
@@ -74,12 +63,9 @@ public function setGuardsProvider(array $guardsProvider)
         $this->guardsProvider = $guardsProvider;
     }
 
-    /**
-     * @return MessagesOptions
-     */
     public function getMessagesOptions(): MessagesOptions
     {
-        if (!$this->messagesOptions) {
+        if (! $this->messagesOptions) {
```

Comment:
```suggestion
        /** @psalm-suppress RedundantPropertyInitializationCheck */
        if (empty($this->messagesOptions)) {
```
We should use `empty($this->messagesOptions)` or better `! isset($this->messageOptions)`
instead of `! $this->messageOptions` in order to avoid the following error thrown by `ForbiddenHanderTest`:

> DotTest\Rbac\Guard\Middleware\ForbiddenHandlerTest::testProcessForbiddenException
> Failed asserting that exception of type "Error" matches expected exception "Dot\Authorization\Exception\ForbiddenException". Message was: "Typed property Dot\Rbac\Guard\Options\RbacGuardOptions::$messagesOptions must not be accessed before initialization" at